bin/lib/amazon.py

```diff
@@ -386,3 +391,51 @@ def has_bouncelock_file(cfg: Config):
         return True
     except s3_client.exceptions.NoSuchKey:
         return False
+
+
+def notify_file_name():
+    return 'ce-notify-file'
+
+
+def has_notify_file():
+    try:
+        s3_client.get_object(
+            Bucket='compiler-explorer',
+            Key=notify_file_name()
+        )
+        return True
+    except s3_client.exceptions.NoSuchKey:
+        return False
+
+
+def put_notify_file(body: str):
+    s3_client.put_object(
+        Bucket='compiler-explorer',
+        Key=notify_file_name(),
+        Body=body,
+        ACL='public-read',
```

Comment:
Does it need a public read and cache control thing? those are only for things that S3 serves as a webpage I think?

Reply:
This is a copy paste fdrom the other "create this file" functions so it most probably needs tweaking yes

Reply:
Removed the CacheControls, left ACL as I'm not too sure if any other available option fits better